src/get-started/create-service.md

```diff
@@ -626,192 +639,60 @@ The code we are going to write is logically separate from the service itself.
 The service library could be connected to an Exonum-powered blockchain
 together with other services,
 while the demo blockchain is a specific example of its usage. For this reason,
-we will position the blockchain code as an [*example*][cargo-example] and
+we will position the blockchain code as an [example][cargo-example] and
 place it into [`examples/demo.rs`][demo.rs].
 
 ### Additional Dependencies
 
 Since services themselves do not require the Exonum node, in this example we
-want to create one, and interact with it as well. Thus, we have to add several
-additional dependencies in our `Cargo.toml`:
+want to create one, and interact with it as well. Thus, we have to more
```

Comment:
Now "we have to more dependencies" sounds awkward :)

Reply:
I'd suggest to either use "we have to specify more dependencies" or "we have one more dependency".